spec.html

```diff
@@ -27168,7 +27168,7 @@ <h1>Example Cyclic Module Record Graphs</h1>
       <emu-clause id="sec-source-text-module-records">
         <h1>Source Text Module Records</h1>
 
-        <p>A <dfn id="sourctextmodule-record" variants="Source Text Module Records">Source Text Module Record</dfn> is used to represent information about a module that was defined from ECMAScript source text (<emu-xref href="#sec-ecmascript-language-source-code"></emu-xref>) that was parsed using the goal symbol |Module|. Its fields contain digested information about the names that are imported by the module and its concrete methods use this digest to link, link, and evaluate the module.</p>
+        <p>A <dfn id="sourctextmodule-record" variants="Source Text Module Records">Source Text Module Record</dfn> is used to represent information about a module that was defined from ECMAScript source text (<emu-xref href="#sec-ecmascript-language-source-code"></emu-xref>) that was parsed using the goal symbol |Module|. Its fields contain digested information about the names that are imported by the module and its concrete methods use this digest to load, link, and evaluate the module.</p>
```

Comment:
I don't think that any of the fields on Source Text Module Records participate in "loading", since the module is already loaded by the time this record is created. However, we can still improve this by mentioning that the exports also participate in linking.

```suggestion
        <p>A <dfn id="sourctextmodule-record" variants="Source Text Module Records">Source Text Module Record</dfn> is used to represent information about a module that was defined from ECMAScript source text (<emu-xref href="#sec-ecmascript-language-source-code"></emu-xref>) that was parsed using the goal symbol |Module|. Its fields contain digested information about the names that are imported and exported by the module, and its concrete methods use these digests to link and evaluate the module.</p>
```

Reply:
@michaelficarra:
> I don't think that any of the fields on Source Text Module Records participate in "loading", since the module is already loaded by the time this record is created.

A Module Record must exist before `FinishLoadingImportedModule` is called on it, indicating that the Module Record is created before or during loading, not after loading. (In particular, a Source Text Module Record is created in `ParseModule`, which looks like it could be invoked well before any module-loading.)

Moreover, `FinishLoadingImportedModule` indirectly invokes `InnerModuleLoading`, which consults the fields of the Module Record to load the module's dependency graph (which is ostensibly part of loading the module itself). So fields in a Source Text Module Record *do* participate in loading the module.

However, I think it's true that the fields and concrete methods that are *specific* to Source Text Module Records (i.e., the ones that are additional to those of a Cyclic Module Record) aren't used during loading.

So I think I'm okay with your suggested change, just not with the reason you gave.